src/components/Nav.svelte

```diff
@@ -134,4 +134,5 @@
         </div>
     </nav>
 </div>
-
+<a href="{`sk/${pathWithoutLang}`}" class="uppercase block md:pr-4"
+               class:selected="{lang === 'sk' ? 'selected' : ''}">{flag('sk')}</a>
```

Comment:
This should be in the src/component/LanguageSwitcher.svelte, else it's floating somewhere on the site

Reply:
Repaired. Switcher placed in src/component/LanguageSwitcher.svelte